src/services/playback.js

```diff
@@ -70,6 +70,8 @@ exports.Playback = class Playback {
                     return this.playNextSong()
                 }
                 if (!res.body.context && this.savedContext) {
+                    // Play saved context only once
+                    this.savedContext = null
                     return this.playSavedContext()
```

Comment:
`this.playSavedContext()` needs to read `this.savedContext` in order to play something. Now it tries to play `null`. Maybe `this.savedContext = null` should be done after the spotify API request has gone through?

Reply:
sorry, that was a rookie mistake. but now it's fixed – and this time I actually tested that it works!